extensions/virtuallistview/src/VirtualListView.tsx

```diff
@@ -1193,27 +1202,30 @@ export class VirtualListView<ItemInfo extends VirtualListViewItemInfo>
         }
     }
 
-    private _selectSubsequentItem(direction: FocusDirection, retry = true) {
+    // Returns true if successfully found/focused, false if not found/focused
+    private _selectSubsequentItem(direction: FocusDirection, retry = true): boolean {
         let index = _.findIndex(this._navigatableItemsRendered, item => item.key === this.state.lastFocusedItemKey);
 
         if (index !== -1 && index + direction > -1 && index + direction < this._navigatableItemsRendered.length) {
             let newElementForFocus = this.refs[this._navigatableItemsRendered[index + direction].vc_key] as VirtualListCell;
             newElementForFocus.focus();
-            return;
+            return true;
         }
 
         if (index === -1 && retry) {
             index = this._itemMap[this.state.lastFocusedItemKey];
 
             if (index === undefined) {
                 assert.ok(false, 'Something went wrong in finding last focused item');
-                return;
+                return false;
```

Comment:
nit: as I understand `return` needs here to avoid TS error? `assert` throws an error, and a code will not continue execution - https://codesandbox.io/s/383xk41nrm

> The [throw](throw https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw) statement throws a user-defined exception. Execution of the current function will stop (the statements after throw won't be executed), and control will be passed to the first catch block in the call stack. If no catch block exists among caller functions, the program will terminate.

Reply:
Typescript doesn't understand that assert throws.  Some projects also choose to strip asserts - this ensures graceful fallback in those cases

Reply:
Got it. Thanks.